### PR TITLE
fix: Fix escaping issues in labels, add fuzz testing to make sure (DEV-3976)

### PR DESCRIPTION
**Pull request by siers**
## Pull Request Checklist

1. Found bug, it looks like it could be old escaping code. The fix has been tested with really nasty output / fuzz testing in OntologyV2R2RSpec. If someone could vouch for the correctness, I'd be grateful.
2. First commit silences the wall of text about slow local queries only for tests.

### Task Description/Number

Issue Number: DEV-

### PR Type

- [ ] build/chore: maintenance tasks (no production code change)
- [ ] docs: documentation changes (no production code change)
- [ ] feat: represents new features
- [x] fix: represents bug fixes
- [ ] perf: performance improvements
- [ ] refactor: represents production code refactoring
- [ ] test: adding or refactoring tests (no production code change)
- [ ] deprecated: Deprecation warning (ideally referencing a migration guide)

### Basic requirements for bug fixes and features

- [ ] Tests for the changes have been added
- [ ] Docs have been added / updated

### Does this PR introduce a breaking change?

- [ ] Yes

### CODE REVIEW

**Review by BalduinLandolt — APPROVED** [review]
NIce! The test seems brute force - but makes sense to me

**Comment by siers** [comment]
@BalduinLandolt fuzz testing, like a stress test, but for security. :)

File: webapi/src/main/scala/org/knora/webapi/messages/v2/responder/resourcemessages/ResourceMessagesV2.scala [hunk 1]
```diff
@@ -674,7 +674,7 @@ object CreateResourceRequestV2 {
   ] =
     ZIO.serviceWithZIO[StringFormatter] { implicit stringFormatter =>
       val validationFun: (String, => Nothing) => String =
-        (s, errorFun) => Iri.toSparqlEncodedString(s).getOrElse(errorFun)
+        (s, errorFun) => s // Iri.toSparqlEncodedString(s).getOrElse(errorFun)
```
Comment (mpro7) [inline webapi/src/main/scala/org/knora/webapi/messages/v2/responder/resourcemessages/ResourceMessagesV2.scala:677]: Should this commented out code be removed?

**Review by mpro7 — APPROVED** [review]

File: webapi/src/main/scala/org/knora/webapi/messages/v2/responder/resourcemessages/ResourceMessagesV2.scala (hunk 1 above)
Comment (siers) [reply webapi/src/main/scala/org/knora/webapi/messages/v2/responder/resourcemessages/ResourceMessagesV2.scala:677]: Yes, absolutely, one second.

Comment (siers) [reply webapi/src/main/scala/org/knora/webapi/messages/v2/responder/resourcemessages/ResourceMessagesV2.scala]: Removed, inlined where necessary, left the original untouched.

**Pull request merged**